recipes/lodepng/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+class LodepngConan(ConanFile):
+    name = "lodepng"
+    description = "PNG encoder and decoder in C and C++, without dependencies."
+    license = "ZLIB"
```

Comment:
```suggestion
    license = "Zlib"
```

https://spdx.org/licenses/Zlib.html